po/zh-Hans.po

```diff
@@ -2031,14 +2039,20 @@ msgid ""
 "* Rust has hygienic macros, `println!` is an example of this.\n"
 "* Rust strings are UTF-8 encoded and can contain any Unicode character."
 msgstr ""
+"*函数以“fn”开头。\n" 
+"*像C和c++一样，块由花括号分隔。\n" 
+"* main函数是程序的入口。\n" 
+"* Rust有卫生宏(hygienic macros)，' println!’就是一个例子。\n" 
+"* Rust字符串是UTF-8编码的，可以包含任何Unicode字符。"
```

Comment:
Add space between Chinese and English may have better display effects in markdown. And `xxx` is a code block in markdown, we'd better not modify it, and we need space between "*" and words which represents a list rather than _italic_
```suggestion
"* 函数以 `fn` 开头。\n" 
"* 像 C 和 C++ 一样，块由花括号分隔。\n" 
"* main 函数是程序的入口。\n" 
"* Rust 有卫生宏 (hygienic macros)，`println!` 就是一个例子。\n" 
"* Rust 字符串是 UTF-8 编码的，可以包含任何 Unicode 字符。"
```